_posts/2019-07-21-introducing-rico.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  'Introducing Rico'
+author: markus
+featuredImage: 
+excerpt: 'Introduction to Rico, the application framework.'
+permalink: '/rico/2019/07/16/introducing-rico.html'
+categories: [Java, Jakarta EE, Java EE, Spring, Rico, Remoting, Monitoring, Security, Angular, WebComponents, JavaFX, Projector]
+header:
+  text: Introducing <span class="my-karakun">Rico<span>
+  image: post
+---
+
+[Rico](/rico) is an application framework fostered by <span class="my-karakun">Karakun</span> to help building better solutions in less time.
+
+Since [Hendrik blogged recently](/java/rico/2019/01/15/rico-server-timing.html) about a in-depth topic on [Rico](/rico), I thought we should give it an introduction post since it has not surfaced much so far....
+
+## History
+
+[Rico](/rico) originates from the _Canoo Dolphin Platform_. 
+As practially everyone who was involved with the development of the _Dolphin Platform_ is working for Karakun now, Canoo shows no interest in it anymore and we still see value in this project, we decided to fork it.
+The result is the [Rico Application Platform](/rico).
+
+## Overview
+
+[Rico](/rico) is build such it can be used on top of the most commonly used frameworks, Spring and Jakarte EE (formerly Java EE) used to build applications based on Java technology nowadays.
+It provides modules for common use cases that extend feature you already get from the framework underneath and simplify access and usage to functionality in question. 
+
+![Rico Modules Overview]({{ "/assets/posts/2019-07-21-introducing-rico/rico-overview.png" | absolute_url }})
+ 
+These are in regard to core functionalities:
+
+* _Data Access_ -
+* _Logging_ - 
+* _Monitoring_ -
+* _Security_ - 
+
+(**TODO:** some word to each module)
+
+Also, as providing a user interface is required for almost every solution in some sort, be it a admin UI or a UI to give users acess the core use case provided, Rico provides some modules to help in that regard.
+
+We see the landscape of UI technologies changing much faster than that of core technology. So Rico provides with __[Rico](/rico) Remoting__ a module that helps to seperate business logic from UI technology and even from client technology (e.g. browser or desktop).
+This separation protects the investment the is placed by building a solution as it makes it much cheaper to reimplement the UI with a new UI technology while keeping the core of the solution as it is (being in production and thus deeply validated by that time).
+Client UI technologies currently supported by adapters are:
+
+* JavaFX
+* Angular
+* AngularJS
+* Polymer (WebComponents)
+
+On top of _[Rico](/rico) Remoting_, there is __[Rico](/rico) Projector__. This module provides a server-side API to create client UIs (currently just JavaFx). 
+It has been initiated and contributed by <a href="https://twitter.com/johado78">Jörg Hälker</a> at <a href="https://www.sprouts.aero/">sprouts GmbH</a> to whom we want to send a big THANK YOU.
```

Comment:
1. why twitter instead of github?
2. why html?

Reply:
he asked for twitter, github is not preferred :)
2) good question. Guess I'll change it ;-)

Reply:
fixed